libraries/chain/contracts/chain_initializer.cpp

```diff
@@ -177,7 +177,7 @@ abi_def chain_initializer::eos_contract_abi(const abi_def& eosio_system_abi)
    eos_abi.structs.emplace_back( struct_def {
       "pending_recovery", "", {
          {"account",    "name"},
-         {"request_id", "uint32"},
+         {"request_id", "checksum256"},
```

Comment:
I think this should be transaction_id_type and we should add the typedef in contracts/eoslib/types.h and the type def in chain_initializer.  The more the multiple definitions match, the easier it will be to keep them consistent, the more people have to look up to verify they are consistent will mean people will just assume they are correct.